xbmc/addons/AddonManager.h

```diff
@@ -138,6 +134,18 @@ namespace ADDON
      */
     bool GetInstalledBinaryAddons(BINARY_ADDON_LIST& binaryAddonList);
 
+    /*!
+     * @brief To get requested installed binary addon on Kodi
+     *
+     * This function becomes used from ADDON::CBinaryAddonManager to get his
+     * related addon (whether enabled or disabled).
```

Comment:
This function is used by ADDON::CBinaryAddonManager to obtain the add-on with the given id, regardless the add-on is disabled or enabled.